homeassistant/components/zha/device_entity.py

```diff
@@ -88,6 +90,9 @@ def device_state_attributes(self):
             del self._device_state_attributes['last_seen']
         self._device_state_attributes['lqi'] = device.lqi
         self._device_state_attributes['rssi'] = device.rssi
+        self._device_state_attributes[QUIRK_APPLIED] = device.quirk_applied
```

Comment:
What's the use case for having the quirk status as state attribute?

Reply:
I need it on the screen to help with support. Is there another way to get info into the more info card?

Reply:
We could just add debug logs.

Reply:
Not really user friendly. I have an idea. I’m going to pull the entity part from the PR but leave the rest. I can handle the display in the config panel. That work?

Reply:
Yes, if it's not part of the state, it's ok.

Reply:
done